core/src/main/scala/kafka/server/Server.scala

```diff
@@ -69,13 +69,7 @@ object Server {
   ): KafkaMetricsContext = {
     val contextLabels = new java.util.HashMap[String, Object]
     contextLabels.put(ClusterIdLabel, clusterId)
-
-    if (config.usesSelfManagedQuorum) {
-      contextLabels.put(NodeIdLabel, config.nodeId.toString)
-    } else {
-      contextLabels.put(BrokerIdLabel, config.brokerId.toString)
```

Comment:
Please remove `BrokerIdLabel` and `MockMetricsReporter.BROKERID`

Reply:
addressed it :)